release-notes.md

```diff
@@ -1,14 +1,14 @@
 # Release 1.9.6
 
 ## What's New
-* none
+* [#378](https://github.com/openziti/desktop-edge-win/issues/378) Cleanup old ziti network adapter profiles
 
 ## Other changes:
-* updated installation config
+* updated auto-installation config
 * renewed signing cert
 
 ## Bugs fixed:
-* none
+* set lower metric, if dns server property is set
```

Comment:
i would put this in 'other changes'. i don't think it's a bug personally but - up to you :)